onnxruntime/python/tools/transformers/optimizer.py

```diff
@@ -214,6 +214,9 @@ def _parse_arguments():
     parser.add_argument('--only_onnxruntime', required=False, action='store_true', help="optimized by onnxruntime only")
     parser.set_defaults(only_onnxruntime=False)
 
+    parser.add_argument('--only_offline_opt', required=False, action='store_true', help="optimized by offline fusion only")
```

Comment:
--only_offline_opt is not needed. If you use "--opt_level 0", that means you will disable runtime optimization.

Reply:
I see we run https://github.com/microsoft/onnxruntime/blob/e7d7fa8fa2c85f5b34bfb4c67f0dc181c5d43dc1/onnxruntime/python/tools/transformers/optimizer.py#L289

it in case of BertOnnxModel, this option can be used in case.

Reply:
Suggest to rename to '--disable_onnxruntime' as discussed.

BTW, our handling of opt_level is not ideal. It has two controls: one is from model_type and another is --opt_level. Maybe we can consolidate them later.